app/Http/Controllers/BookController.php

```diff
@@ -68,7 +76,17 @@ public function store(BookFormRequest $request)
      */
     public function show($id)
     {
-        //
+        try {
+            $book = Book::findOrFail($id);
+            $selectedCategories = $book->categories()->get();
```

Comment:
```suggestion
            $selectedCategories = $book->categories;
```
sử dụng như này cho gọn nhé